File: task-03/elixir/primen.ex
IO.puts("Enter a number:")
x = String.to_integer(IO.gets(""))

for i <- 2..x do
  f = 0

  for j <- 2..(i-1) do
    if rem(i, j) == 0 do
      f = 1
      
    end
  end

  if f == 0 do
    IO.puts("#{i} : Prime")
  end
end

File: task-03/ruby/primen.rb
print "Enter the number: "
x = gets.chomp.to_i

(2..x).each do |i|
    f = 0
    (2..i-1).each do |j|
        if i % j == 0
            f = 1
            break
        end
    end

    if f == 0
        puts "#{i}: prime"
    end
end
